#goal definition
bool runRequested
---
#result definition
bool tabletConnected
bool cloudFault
---
#feedback
bool stow
bool deploy
bool teleopDrive
bool EStop
bool connectedEnabled
bool cancelAutoDrive
bool deployPiLit
bool retrievePiLit
bool deployAllPiLits
float64[] placeLatLong
string formationType
float64[] driveToLatLong
bool retrieveAllPiLits

